addons/mail/static/src/components/notification_popover/notification_popover.xml

```diff
@@ -3,13 +3,15 @@
 
     <t t-name="mail.NotificationPopover" owl="1">
         <div class="o_NotificationPopover" t-attf-class="{{ className }}" t-ref="root">
-            <t t-foreach="notifications" t-as="notification" t-key="notification.localId">
-                <div class="o_NotificationPopover_notification">
-                    <i class="o_NotificationPopover_notificationIcon" t-att-class="iconClass" t-att-title="iconTitle" role="img"/>
-                    <t t-if="notification.partner">
-                        <span class="o_NotificationPopover_notificationPartnerName" t-esc="notification.partner.nameOrDisplayName"/>
-                    </t>
-                </div>
+            <t t-if="message">
+                <t t-foreach="message.notifications" t-as="notification" t-key="notification.localId">
+                    <div class="o_NotificationPopover_notification">
+                        <i class="o_NotificationPopover_notificationIcon" t-att-class="iconClass" t-att-title="iconTitle" role="img"/>
```

Comment:
Do I miss something or `iconClass` and `iconTitle` are broken?

They use `this.notification` in the JS, do they even have access to the `foreach` variable?

It seems that is at least very weak and potential to be broken (in particular with Owl 2).

We should probably move those getters as field into the notification record immediately.